analyticsdataserver/tests/test_clients.py

```diff
@@ -0,0 +1,95 @@
+import json
```

Comment:
This should be the same as the tests that were in `analyticsdataserver/tests.py`.